tests/acceptance/features/cliCommands/uploadSessions.feature

```diff
@@ -76,7 +76,7 @@ Feature: List upload sessions via CLI command
     And the config "STORAGE_USERS_UPLOAD_EXPIRATION" has been set to "0"
     And user "Alice" has uploaded file with content "upload content" to "/file2.txt"
     And user "Alice" has uploaded file with content "upload content" to "/file3.txt"
-    When the administrator cleans all expired upload sessions
+    When the administrator cleans upload sessions with flag "expired"
```

Comment:
could use table data because there can be multiple flags
```suggestion
    When the administrator cleans upload sessions with the following flags:
      | expired |
```